### PR TITLE
Changes according to PR #1

### PR DESCRIPTION
**Pull request by dankwart-de**
Don't worry, seems much but isn't.

### CODE REVIEW

File: src/de/dkfz/b080/co/common/BasicCOProjectsRuntimeService.groovy [hunk 1]
```diff
@@ -40,13 +42,13 @@ public class BasicCOProjectsRuntimeService extends RuntimeService {
...
-                (COConstants.PRM_CONFIG_FILE)    : fs.getNameOfConfigurationFile(context).getAbsolutePath(),
-                (COConstants.PRM_ANALYSIS_DIR)   : context.getOutputDirectory().getParentFile().getParent()
+        Map<Object, Object> parameters = [
+                (COConstants.PRM_PID)         : (Object) pid,
+                (COConstants.PRM_PID_CAP)     : pid,
+                (COConstants.PRM_CONFIG_FILE) : fs.getNameOfConfigurationFile(context).getAbsolutePath(),
+                (COConstants.PRM_ANALYSIS_DIR): context.getOutputDirectory().getParentFile().getParent()
         ]
-        return parameters;
+        return parameters as Map<String, String>;
```
Comment (vinjana) [inline src/de/dkfz/b080/co/common/BasicCOProjectsRuntimeService.groovy]: Only a small thing, but  would this code not be clearer, if we used `(String) pid` and `getParent().toString()` and leave out the _post hoc_ type cast?

Comment (dankwart-de) [reply src/de/dkfz/b080/co/common/BasicCOProjectsRuntimeService.groovy]: Good point, I'll change it.

**Comment by dankwart-de** [comment]
As we discussed, I'd create a new Plugin, which we could call COWorkflows. We could then put in all the removed scripts and not break compatibility. How about that? Don't know yet, where to git/host it though.

**Comment by vinjana** [comment]
But, if you keep the scripts that you have removed here, why did you remove them here in the at all? What is the advantage to separate the Groovy/Java code from the Bash code in different plugins? (And BTW, are there functions in the Groovy code that call Bash scripts? These I would not move!)

**Comment by dankwart-de** [comment]
The problem is, that I do not know, who wrote which script and how they can be published. The COWorkflows Plugin would be for internal use, whereas the proposed form would only keep non-problematic parts.

**Comment by dankwart-de** [comment]
Well I could just ask around a few people. Still I don't know who wrote what.

**Comment by vinjana** [comment]
O.k. -- now I get it. Good reason. Can we call the new plugin something else than a previously existing plugin? Maybe "COWorkflowInternal"?

**Comment by dankwart-de** [comment]
Closed because of #4 
This is a corrected version branched off of the proper base. Also at least some of the files are checked for usage.

**Pull request closed without merging**